engine/execution/computation/computer/computer.go

```diff
@@ -456,6 +458,9 @@ func (e *blockComputer) executeTransaction(
 	res.AddServiceEvents(tx.ServiceEvents)
 	res.AddTransactionResult(&txResult)
 	res.AddComputationUsed(tx.ComputationUsed)
+	if tx.IsSampled() {
```

Comment:
This condition probably isn't necessary. Just make sure `MergeComputationEffortVector` can handle nil values.